examples/foomedical/src/pages/ScreeningQuestionnairePage.tsx

```diff
@@ -0,0 +1,2412 @@
+import React from 'react';
+import { getQuestionnaireAnswers } from '@medplum/core';
+import { Questionnaire, QuestionnaireResponse } from '@medplum/fhirtypes';
+import { Document, QuestionnaireForm } from '@medplum/react';
+
+async function handleQuestionnaireSubmit(formData: QuestionnaireResponse): Promise<void> {
+  const answers = getQuestionnaireAnswers(formData);
+  console.log(answers);
```

Comment:
should we do something else once form is submitted ?

Reply:
Good question.... Maybe for the sake of demo, we can show some kind of "Thank you for submitting" message?  I don't think we need to persist the results anywhere, as this is just for a demo.